source/exceptions.tex

```diff
@@ -210,8 +210,8 @@
 can be caught by a
 \grammarterm{handler}
 of
-\tcode{const}
-\tcode{char*}
+\keyword{const}
+\tcode{\keyword{char}*}
```

Comment:
Above `char` remains with `\tcode`. I agree with @jwakely with regards to `\libconcept` that this makes reviews harder. Have you considered simply using `\tcode` and `\codeblock` to delimit C++ code and use a custom tool that adds all the bells and whistles to the LaTeX?

Reply:
Huh? It does not. This change adds `\keyword` around `char`.

I do agree that adding \libconcept makes reading the source harder for codeblocks, but this is actually outside of a codeblock. (For listings = codeblocks, there is already keyword autodetection; maybe we can also have concept autodetection there and add \libconcept automatically.)